CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Fixed
 
 - xapi: be sure to send completed event when progress has reached 100%
+- front: add a resolution for js-yaml package to avoid using a version
+  with a security risk
+- aws: add a resolution for handlebars package to avoid using a version
+  with a security risk
```

Comment:
There is also a `Security` sub-section in the Keep a changelog standard https://keepachangelog.com/en/1.0.0/

Reply:
Thanks I changed it.